Makefile

```diff
@@ -128,6 +128,7 @@ dev:
 		&& rm -rf /Volumes/RAMDisk/mist \
 		&& export TMP=/Volumes/RAMDisk; \
 	fi \
+	&& export PATH=$$PATH:$$(pwd)/bin \
```

Comment:
i don't know if the `PATH` gets exported to the shell calling make commands or is it only for make's context. I'd avoid modifying system's `PATH` value here